### DISCUSSION TITLE
.NET upgrade

### DISCUSSION THREAD

**Discussion by pugaz-bits1n0** [kohsuke/winsw#946] [General]
Any plans to upgrade to the latest .NET version. Asking this with regards to this CVE-2019-0820, impacting the regular expressions library. 

https://msrc.microsoft.com/update-guide/vulnerability/CVE-2019-0820

**Comment by oleg-nenashev** [comment]
The request is about the .NET Core distribution only, right?

**Reply by viceice** [reply]
yes, dotnet 5 is EOL since May 10, 2022

https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

So even the v2 version should get an update to net6 with is a lts release again.

I can provide a PR for v2 if it will be accepted

**Comment by nxtn** [comment]
2.12.0 is now on .NET 6.
